include/System/THPRender.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef SYSTEM_THP_RENDER_HPP
+#define SYSTEM_THP_RENDER_HPP
+
+#include <JSystem/JDrama/JDRViewObj.hpp>
+#include <JSystem/JDrama/JDRGraphics.hpp>
+#include "dolphin/gx/GXTransform.h"
+
+#include "MarioUtil/DrawUtil.hpp"
+#include "THPPlayer/THPPlayer.h"
```

Comment:
I've been using `<>` brackets almost everywhere for absolute include paths, so better to follow the convention